### PR TITLE
[query] Print file size on Table/MT write

### PR DESCRIPTION
**Pull request by tpoterba**
Next, I want to put file sizes into the metadata.json

### CODE REVIEW

File: hail/src/main/scala/is/hail/utils/package.scala [hunk 1]
```diff
@@ -200,18 +200,32 @@ package object utils extends Logging
...
+    } else if (absds < gib) {
+      div = mib
+      suffix = "MiB"
+    } else if (absds < tib) {
+      div = gib
+      suffix = "GiB"
+    } else {
+      div = tib
+      suffix = "TiB"
+    }
```
Comment (chrisvittal) [inline hail/src/main/scala/is/hail/utils/package.scala:226]: Why not
```scala
val (div, suffix) = if (...)
  1 -> "B"
else if (...)
  kib -> "KiB"
// etc.
```

**Review by chrisvittal — CHANGES_REQUESTED** [review]
Minor comments. Seems fine otherwise.

Intention with the metadata is to add another array that is the partition sizes in bytes?

File: hail/python/hail/experimental/vcf_combiner/vcf_combiner.py [hunk 2]
```diff
@@ -321,7 +321,7 @@ def calculate_new_intervals(ht, n, reference_genome):
     ht = ht.annotate(y=ht.x + 1)
     ht = ht.filter((ht.x // n != ht.y // n) | (ht.x == (n_rows - 1)))
     ht = ht.select()
-    ht = ht.annotate(start=hl.or_else(
+    ht = ht.annoatte(start=hl.or_else(
```
[the remark this answers is not in the document]
Comment (tpoterba) [reply hail/python/hail/experimental/vcf_combiner/vcf_combiner.py:324]: huh, wonder how I even made this change!

File: hail/src/main/scala/is/hail/utils/package.scala (hunk 1 above)
Comment (tpoterba) [reply hail/src/main/scala/is/hail/utils/package.scala:226]: wanted to avoid the allocation, but this is probably cleaner.

**Comment by tpoterba** [comment]
good comments, thanks! Addressed.

**Review by chrisvittal — APPROVED** [review]

**Pull request merged**